includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -205,7 +205,8 @@ public function get_client() {
 	 * @return Google_Site_Kit_Client|Google_Site_Kit_Proxy_Client
 	 */
 	private function setup_client() {
-		if ( $this->using_proxy() ) {
+		$credentials = $this->get_client_credentials();
```

Comment:
`get_client_credentials` does not return the same `Credentials` instance that is available via `$this->credentials` which you're already using. This line is safe to remove as it isn't being used.

Reply:
Fixed